src/main/java/com/jcabi/http/request/FakeRequest.java

```diff
@@ -144,10 +145,31 @@ public FakeRequest(final int status,
         @NotNull(message = "list of headers can't be NULL")
         final Collection<Map.Entry<String, String>> headers,
         @NotNull(message = "body can't be NULL") final byte[] body) {
+        this(
+            status,
+            reason,
+            headers,
+            Collections.singletonMap(Pattern.compile(".*"), body.clone())
+        );
+    }
+
+    /**
+     * Public ctor.
+     * @param status HTTP status code to return
+     * @param reason HTTP reason
+     * @param headers HTTP headers
+     * @param bodies HTTP bodies to return
+     * @checkstyle ParameterNumber (10 lines)
+     */
+    public FakeRequest(final int status,
+        @NotNull(message = "HTTP reason can't be NULL") final String reason,
+        @NotNull(message = "list of headers can't be NULL")
+        final Collection<Map.Entry<String, String>> headers,
+        final Map<Pattern, byte[]> bodies) {
         this.code = status;
         this.phrase = reason;
         this.hdrs = new Array<Map.Entry<String, String>>(headers);
-        this.content = body.clone();
+        this.content = bodies;
```

Comment:
@dmzaytsev maybe instead of assigning we should add all map elements to the object's internal map, which would be unmodifiable. what do you think?